drivers/led/lp5569.c

```diff
@@ -30,12 +31,15 @@ LOG_MODULE_REGISTER(lp5569, CONFIG_LED_LOG_LEVEL);
 
 #define LP5569_MISC			0x2F
 #define LP5569_POWERSAVE_EN		BIT(5)
+#define LP5569_CP_MODE_SHIFT		3
 
 /* PWM base Register for controlling the duty-cycle */
 #define LP5569_LED0_PWM			0x16
 
 struct lp5569_config {
 	struct i2c_dt_spec bus;
+	struct gpio_dt_spec en_pin;
+	const uint8_t cp_mode;
```

Comment:
I think that this is preferred. CP is not an abbreviation that I'm familiar with at least, so it doesn't hurt to be more explicit.
Same goes for the macro etc. 
```suggestion
	const uint8_t charge_pump_mode;
```

Reply:
`cp_mode` directly refers to the name provided by the datasheet for the bits this configures, so I prefer it staying as such. I think it's easier for me to argue for when I send these patches upstream.
But I totally see, where you're coming from. For device drivers I think it's reasonable to have to open the datasheet when reading the code.